nBTC/sources/nbtc.move

```diff
@@ -52,31 +67,29 @@ fun init(witness: NBTC, ctx: &mut TxContext) {
         NAME,
         DESCRIPTION,
         option::some(url::new_unsafe_from_bytes(ICON_URL)),
-        ctx
+        ctx,
     );
 
     transfer::public_freeze_object(metadata);
-    let treasury =
-     WrappedTreasuryCap {
-         id: object::new(ctx),
-         cap: treasury_cap,
-         tx_ids: table::new<vector<u8>, bool>(ctx),
-         trusted_lc_id: TRUSTED_LIGHT_CLIENT_ID.to_id(),
-         fallback_address: FALLBACK_ADDRESS,
-         btc_treasury: BTC_TREASURY,
-         };
+    let treasury = WrappedTreasuryCap {
+        id: object::new(ctx),
+        cap: treasury_cap,
+        tx_ids: table::new<vector<u8>, bool>(ctx),
+        trusted_lc_id: TRUSTED_LIGHT_CLIENT_ID.to_id(),
+        fallback_address: FALLBACK_ADDRESS,
+        btc_treasury: BTC_TREASURY,
+    };
     transfer::public_share_object(treasury);
-
 }
 
-//
-// PUBLIC ENTRY FUNCTIONS
-//
```

Comment:
same here - adding extra `//` lines make it more visible

Reply:
Done